example-audit-policy.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: audit.k8s.io/v1beta1
+kind: Policy
+omitStages:
+  - "RequestReceived"
+rules:
+  - level: Request
+    verbs:
+      - create
+      - delete
+      - patch
+    resources:
+      - group: "postgrescontroller.kubeplus"
+        version: "v1"
+        resources: ["postgreses"]
```

Comment:
In the README we are showing the provenance operators applied to 'deployment' paths. But here we are not specifying deployments in the resources. We are actually specifying 'postgreses'. So shouldn't the provenance operators be also applied to 'postgreses' paths?

I think the kind_compositions.yaml need to be updated to include Postgreses. Check the kinds_compositions for etcd (it should be in this repo -- it is also there in kubediscovery). Postgreses is composed of 'Deployment' and 'Service'.